app/views/layouts/admin.html.erb

```diff
@@ -109,7 +109,7 @@
 <% end %>
 
 <% content_for :body_end do %>
-  <%= javascript_include_tag "admin", integrity: true, crossorigin: 'anonymous' %>
```

Comment:
You're only doing it for the JS, but the CSS also has SRI, is it worth removing it from there too?  Maybe not if we've not had any reports about the site not looking right.

Reply:
Yeah... I considered that. The tickets we've had have been specifically about the 'New Document' and 'More' tabs not working with no mention of styling so I left it on there on the assumption that it isn't causing any problems.

Reply:
:+1: